env/src/lib.rs

```diff
@@ -337,7 +356,12 @@ impl Log for Logger {
             }
         }
 
-        let _ = writeln!(&mut io::stderr(), "{}", (self.format)(record));
+        match self.target {
+            LogTarget::Stdout => println!("{}", (self.format)(record)),
```

Comment:
`writeln!` with `io::stdout` is not being captured by the test harness according to my tests.